app/home.html

```diff
@@ -1,18 +1,9 @@
 <!doctype html>
 <html dir="ltr">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
-    <title>MetaMask</title>
-    <link rel="preload" href="/_locales/en/messages.json" as="fetch" crossorigin="anonymous" />
-    <link rel="stylesheet" href="../ui/css/index.scss">
+    <%~ include("partial-head.html") %>
   </head>
   <body>
-    <div id="app-content">
-      <img class="loading-logo" src="./images/logo/metamask-fox.svg" alt="" loading="lazy" />
-      <img class="loading-spinner" src="./images/spinner.gif" alt="" loading="lazy" />
-    </div>
-    <div id="popover-content"></div>
-    <script src="./scripts/load/ui.ts" defer></script>
+    <%~ include("partial-body.html") %>
   </body>
-</html>
+</html>
```

Comment:
@davidmurdoch you have something misconfigured here and our lint really should be catching it as well, but maybe it's ignoring html files in the app folder?

All files should end with a blank line.  You can see in GitHub that 4 of these html files end with ⛔ (colors are different) which means they do not end in a blank line.

Reply:
Yeah, if you hit save in VSCode on any of our HTML files they drastically change; I usually try to "Save without formatting" to avoid it. If you have a fix i'd love to have it!

Reply:
Oh I see, you're deliberately skipping lint.  Maybe we should just accept the formatting change?

Reply:
A trailing newline is enforced by Prettier, but we don't process HTML files with Prettier.

For HTML, the EditorConfig plugin should ensure your editor defaults to using a trailing newline. But maybe you don't have that installed.